ipc/identity/src/evm/mod.rs

```diff
@@ -53,6 +57,12 @@ impl Drop for KeyInfo {
     }
 }
 
+/// This trait is use to determine the key chosen for a specific
+/// key in a general way.
+pub trait Defaultable {
```

Comment:
My first impression was `Default` for general types, but it happens to be about default key. Also returning `Self` implies `Self` is a `key`?

Reply:
fixed.